### PR TITLE
:tada: Eva onboarding team up

### PR DESCRIPTION
**Pull request by EvaMarco**
This PR solves this https://tree.taiga.io/project/penpot/us/3863?milestone=322427

### CODE REVIEW

File: frontend/src/app/main/ui/dashboard/projects.cljs [hunk 1]
```diff
@@ -196,11 +214,17 @@
...
-                        (sort-by :modified-at)
-                        (reverse))
-        recent-map (mf/deref recent-files-ref)]
+  [{:keys [team projects profile] :as props}]
+  (let [projects    (->> (vals projects)
+                         (sort-by :modified-at)
+                         (reverse))
+        recent-map  (mf/deref recent-files-ref)
+        props       (some-> profile (get :props {}))
+        team-hero? (:team-hero? props true)
```
Comment (superalex) [inline frontend/src/app/main/ui/dashboard/projects.cljs]: Extra space here? :)

File: frontend/src/app/main/ui/dashboard/team.cljs [hunk 2]
```diff
@@ -34,7 +34,7 @@
   (let [go-members           (mf/use-fn #(st/emit! (dd/go-to-team-members)))
         go-settings          (mf/use-fn #(st/emit! (dd/go-to-team-settings)))
         go-invitations       (mf/use-fn #(st/emit! (dd/go-to-team-invitations)))
-        invite-member        (mf/use-fn #(st/emit! (modal/show {:type :invite-members :team team})))
+        invite-member        (mf/use-fn #(st/emit! (modal/show {:type :invite-members :team team :origin :team})))
```
Comment (niwinz) [inline frontend/src/app/main/ui/dashboard/team.cljs]: missing `team` in hook deps

File: frontend/src/app/main/ui/dashboard/projects.cljs (hunk 1 above)
Comment (EvaMarco) [reply frontend/src/app/main/ui/dashboard/projects.cljs]: :) changed¡

File: frontend/src/app/main/ui/dashboard/team.cljs (hunk 2 above)
Comment (EvaMarco) [reply frontend/src/app/main/ui/dashboard/team.cljs]: Good point, Added¡¡

**Pull request merged**